TreeTracker/UI/Views/Profile Screen/ProfileViewController.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  ProfileViewController.swift
+//  TreeTracker
+//
+//  Created by Remi Varghese on 3/23/21.
+//  Copyright © 2021 Greenstand. All rights reserved.
+//
+
+import UIKit
+
+class ProfileViewController: UIViewController {
+    var trees: [Tree] = []
+    weak var rootcoordinator: RootCoordinator!
+    var homecoordinator: HomeCoordinator!
+    // MARK: - IBOutlets
+    @IBOutlet weak var profilepic: UIImageView!
+    @IBOutlet private var nameLabel: UILabel! {
+        didSet {
+            nameLabel.font = FontFamily.Montserrat.semiBold.font(size: 16.0)
+        }
+    }
+    @IBOutlet private var emailphoneLabel: UILabel! {
+           didSet {
+               emailphoneLabel.font = FontFamily.Montserrat.semiBold.font(size: 16.0)
+           }
+    }
+    @IBOutlet private var organizationLabel: UILabel! {
+        didSet {
+            organizationLabel.font = FontFamily.Montserrat.semiBold.font(size: 16.0)
+        }
+    }
+    @IBAction func changeuserButton(_ sender: UIButton) {
```

Comment:
Should we make this function private so it can't be accessed from outside the view?

Reply:
Also the function name could be a little more descriptive - `changeUserButtonPressed()` maybe? Keep an eye on the capitalisation.

As we're not using the sender value either maybe we should remove it?

Reply:
Made the changes in @IBAction func